demos/button.html

```diff
@@ -66,345 +80,116 @@
     <main class="mdc-toolbar-fixed-adjust">
 
       <section class="hero">
-        <button class="mdc-button button">Flat</button>
-        <button class="mdc-button mdc-button--raised button mdc-button--primary">Raised</button>
+        <button class="mdc-button button">
```

Comment:
Good catch, I didn't notice this old bug. I think you are right, this is not being used anywhere.